src/components/ui/Slider/Slider.js

```diff
@@ -0,0 +1,34 @@
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/css';
+import '../../../index.css';
+import { Navigation } from 'swiper/modules';
+import arrowLeft from '../../../icons/arrowLeft.svg';
+import arrowRight from '../../../icons/arrowRight.svg';
+import { cn } from '../../../lib/utils';
+
+export const Slider = ({ spaceBetween, slidesPerView, className, children }) => {
+  return (
+    <>
+      <Swiper
+        className={cn('w-full h-full', className)}
+        navigation={{
+          nextEl: '.image-swiper-button-next',
+          prevEl: '.image-swiper-button-prev',
+          disabledClass: 'swiper-button-disabled'
+        }}
+        modules={[Navigation]}
+        spaceBetween={spaceBetween}
+        slidesPerView={slidesPerView}
+        onSlideChange={() => console.log('slide change')}
+        onSwiper={(swiper) => console.log(swiper)}>
+        <SwiperSlide className="text-white">{children}</SwiperSlide>
```

Comment:
this will only make 1 slide, you need to use Children.map

smth like:

`

{Children.map(children, (child, index) => (
            <SwiperSlide key={index}>{child ?? ''}</SwiperSlide>
          ))}`